.github/pull_request_template.md

```diff
@@ -14,10 +14,19 @@ and code of conduct https://github.com/grafana/k6/blob/master/CODE_OF_CONDUCT.md
 -->
 
 - [ ] I have performed a self-review of my code.
-- [ ] I have added tests for my changes.
-- [ ] I have run linter locally (`make lint`) and all checks pass.
-- [ ] I have run tests locally (`make tests`) and all tests pass.
 - [ ] I have commented on my code, particularly in hard-to-understand areas.
+- [ ] I have added tests for my changes.
+- [ ] I have run linter and tests locally (`make lint` and `make tests`) and all pass.
+
+## Checklist: Documentation (only for k6 maintainers and if relevant)
+
+**Please do not merge this PR until the following items are filled out.**
+
+- [ ] I have added the correct milestone to the PR.
```

Comment:
It's also good to expect that useful labels like breaking change will be added around that time 

```suggestion
- [ ] I have added the correct milestone and labels to the PR.
```